docs/introduction/developer-quickstart.md

```diff
@@ -143,6 +182,17 @@ How it works:
 
 All you have to do is emit events as usual, and the platform applies common-sense visibility rules.
 
+
+# 3. Integrating Obscuro Network in the Frontend
+
+Creating a user-friendly frontend is crucial for interacting with your smart contracts on Obscuro. This section will guide you through installing necessary packages and integrating the Obscuro network.
+
+```bash
+npm install @obscuro/obscuro-gateway-widget
+```
+
+Import and use the package in your components to interact with the Obscuro network.
```

Comment:
can we be a bit more specific here?
What exactly do you have to do after installing the plugin?